addons/sale_stock/tests/test_sale_stock.py

```diff
@@ -841,8 +841,7 @@ def test_14_delivered_qty_in_multistep(self):
         picking_form = Form(delivery)
         with picking_form.move_ids_without_package.edit(0) as move:
             move.quantity = 10
-        with picking_form.move_ids_without_package.new() as new_move:
-            new_move.product_id = product_inv_on_order
+        with picking_form.move_ids_without_package.edit(1) as new_move:
```

Comment:
move automatically propagated from the previous picking so just set the quantity instead of creating another move